ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -243,14 +249,17 @@ const ConfirmPageContainer = (props) => {
                         {t('buyAsset', [nativeCurrency])}
                       </Button>,
                     ])}
-                  </Typography>
+                  </Text>
                 ) : (
-                  <Typography variant={TypographyVariant.H7} align="left">
+                  <Text
+                    variant={TextVariant.bodySm}
+                    textAlign={TEXT_ALIGN.LEFT}
+                  >
                     {t('insufficientCurrencyDeposit', [
                       nativeCurrency,
                       networkName,
                     ])}
-                  </Typography>
+                  </Text>
                 )
               }
```

Comment:
Not ideal because it's still not a certainty but what I did to do a visual check is take the local keys and mock it up in storybook using the ActionableMessage component. I think this would be sufficient for a visual check


```
// in ui/components/ui/actionable-message/actionable-message.stories.js

export const ConfirmPageContainer = () => (
  <ActionableMessage
    message={
      <>
        <Text variant={TextVariant.bodySm} textAlign={TEXT_ALIGN.LEFT}>
          You do not have enough $1 in your account to pay for transaction fees
          on $2 network. $3 or deposit from another account.
          <Button
            type="inline"
            className="confirm-page-container-content__link"
            onClick={() => setShowDepositPopover(true)}
          >
            Buy $1
          </Button>
          ,
        </Text>
        <Text variant={TextVariant.bodySm} textAlign={TEXT_ALIGN.LEFT}>
          You do not have enough $1 in your account to pay for transaction fees
          on $2 network. Deposit $1 from another account.
        </Text>
      </>
    }
    useIcon
    iconFillColor="var(--color-error-default)"
    type="danger"
  />
);
```

<img width="1438" alt="Screenshot 2023-02-15 at 9 30 02 AM" src="https://user-images.githubusercontent.com/8112138/219107225-30d67df7-c7ed-4e91-b9d7-fc9c9e4e3438.png">